README.md

```diff
@@ -78,6 +78,15 @@ Follow the given instructions for Login into the app.
 
 To run the tests run: `python manage.py test`.
 
+## Documentation
+
+- Join us on [Open session](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs/topic/Open.20Sessions) and see [Meeting minutes](https://docs.google.com/document/d/1YF13IbBrU1ln4ZF1fOpgb-xGRgIF6tZLSjIBQgDmN7k/edit).
+- For setting up the project locally watch [this video](https://youtu.be/_b2RQGbYN9w).
+- Checkout the video of [project demonstration](https://youtu.be/3A746GppZ0Y) here.
```

Comment:
One last thing, I think if you used `here` in this line then the link should be linked to that. 
for e.g:
`Check out the video of project demonstration [here](https://youtu.be/3A746GppZ0Y).`

Alternative: You can modify the sentence also by removing the here keyword.